pkg/pgmodel/ingestor/ingestor_sql_test.go

```diff
@@ -55,8 +56,8 @@ func TestPGXInserterInsertSeries(t *testing.T) {
 					Sql: "SELECT * FROM _prom_catalog.get_or_create_label_ids($1, $2, $3)",
 					Args: []interface{}{
 						"metric_1",
-						[]string{"__name__", "name_1"},
-						[]string{"metric_1", "value_1"},
+						&pgsafetype.TextArray{TextArray: pgtype.TextArray{Elements: []pgtype.Text{pgtype.Text{String: "__name__", Status: 0x2}, pgtype.Text{String: "name_1", Status: 0x2}}, Dimensions: []pgtype.ArrayDimension{pgtype.ArrayDimension{Length: 2, LowerBound: 1}}, Status: 0x2}},
```

Comment:
can't you make this conversion from []string. []string to pgsafetype.TextArray in the test code below? This is much less readable.